spotify_player/src/cli/mod.rs

```diff
@@ -59,15 +59,41 @@ pub enum IdOrName {
 #[derive(Debug, Serialize, Deserialize)]
 pub enum Command {
     StartContext(ContextType, IdOrName),
-    StartLikedTracks { limit: usize, random: bool },
+    StartLikedTracks {
+        limit: usize,
+        random: bool,
+    },
     StartRadio(ItemType, IdOrName),
     PlayPause,
     Next,
     Previous,
     Shuffle,
     Repeat,
-    Volume { percent: i8, is_offset: bool },
+    Volume {
+        percent: i8,
+        is_offset: bool,
+    },
     Seek(i64),
+    PlaylistNew {
+        name: String,
+        public: bool,
+        collab: bool,
+        description: String,
+    },
+    PlaylistDelete {
+        id: PlaylistId<'static>,
+    },
+    PlaylistList,
+    PlaylistImport {
+        from: PlaylistId<'static>,
+        to: PlaylistId<'static>,
+    },
+    PlaylistFork {
+        id: PlaylistId<'static>,
+    },
+    PlaylistUpdate {
+        id: Option<PlaylistId<'static>>,
+    },
```

Comment:
I think it should be put into another enum `PlaylistCommand`. The initial enum can be renamed to `PlaybackCommand`.

Reply:
With that, the variant's name can be reduced, for example, `PlaylistNew` to `New` for better clarity.